packages/website/src/examples/Table/main.example.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {Badge, TableHOC, TableRowConnected} from '@coveord/plasma-react';
+import {loremIpsum} from 'lorem-ipsum';
+
+export default () => (
+    <TableHOC
+        id={'tableId'}
```

Comment:
```suggestion
        id="tableId"
```

Or use une variable for `tableId`